public/app/plugins/datasource/prometheus/querybuilder/shared/LokiAndPromQueryModellerBase.ts

```diff
@@ -43,7 +43,10 @@ export abstract class LokiAndPromQueryModellerBase<T extends QueryWithOperations
 
   renderOperations(queryString: string, operations: QueryBuilderOperation[]) {
     for (const operation of operations) {
-      const def = this.operationsRegisty.get(operation.id);
+      const def = this.operationsRegisty.getIfExists(operation.id);
+      if (!def) {
+        throw new Error(`Could not find operation ${operation.id} in the registry`);
```

Comment:
Added more readable error but wonder if it would not make more sense to render some `<OP_NOT_FOUND>` placeholder or something so the user can more easily go back. On the other hand, this should be very rare and if it happens should be easy to fix so maybe making it more probable to get it reported would be better.